packages/lexure/src/lib/parser/Parser.ts

```diff
@@ -0,0 +1,21 @@
+import type { Parameter } from '../lexer/streams/ParameterStream';
+import type { IUnorderedStrategy } from './strategies/IUnorderedStrategy';
+import { ParserResult } from './ParserResult';
+import { EmptyStrategy } from './strategies/EmptyStrategy';
+
+export class Parser {
+	public strategy: IUnorderedStrategy;
+
+	public constructor(strategy: IUnorderedStrategy) {
```

Comment:
```suggestion
	public constructor(strategy?: IUnorderedStrategy) {
```
This is to make the next line valid